presto-hive/src/test/java/com/facebook/presto/hive/TestHiveClientFileMetastore.java

```diff
@@ -66,4 +105,117 @@ public void testTransactionDeleteInsert()
     {
         // FileHiveMetastore has various incompatibilities
     }
+
+    @Test
+    public void testPartitionNotReadable()
+            throws Exception
+    {
+        SchemaTableName tableName = temporaryTable(tempTable);
+        ImmutableMap dynamicPartitionParameters = ImmutableMap.of(
+                HiveSplitManager.OBJECT_NOT_READABLE, "Testing Unreadable Partition");
```

Comment:
- static import OBJECT_NOT_READABLE
- move this together with the previous line